ReSharper.FSharp/src/FSharp.Psi/src/FSharp.psi

```diff
@@ -1448,3 +1451,13 @@ anonType:
 
 otherType:
   fakeSeparator;
+
+staticConstant:
+  fakeSeparator; 
+staticConstantExpr:
+  fakeSeparator;
+
+staticConstantNamed: 
+  IDENTIFIER<IDENTIFIER, Identifier>
```

Comment:
It doesn't seem to correspond to how these nodes are converted in the tree builder. Have you tried accessing the identifier property?
Right now there's a synType conversion which AFAIK never yields a plain identifier here.

We don't have to copy the FCS AST structure completely and instead we should try to make it easier to use it on our side. For instance, look at `SynType.LongIdent` vs `typeReferenceName` structure differences where qualified identifiers are placed as child qualifier nodes and outer type arguments are added right into the reference name node.